model/clusters_mgmt/v1/aws_backup_config_type.model

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Backup configuration for AWS clusters
+struct AWSBackupConfig {
+    // ID of the AWS Disaster Recovery (DR) account
+    AccountID           string
```

Comment:
Do we still use the `FooID` convention or have we moved to `FooId` instead? I think this should be `AccountId`, but not sure if we've adopted that entirely or if it's relegated to just variable names.

Reply:
I couldn't find explicit guidelines around that. In any case, I've updated it to `AccountId`.